app/liquid_tags/liquid_tag_base.rb

```diff
@@ -3,11 +3,39 @@ def self.script
     ""
   end
 
+  def initialize(_tag_name, _content, _parse_context)
+    super
+    validate_contexts
+    # This check issues DB queries so we keep it as the last one
+    Pundit.authorize(
+      parse_context.partial_options[:user],
```

Comment:
I assume this has to do with inheriting from `Liquid::Tag` and calling `super` (assumption: the super class uses the argument and makes it available via an accessor), but it's confusing to see `parse_context` used when we have a param that suggests it isn't (`_parse_context`).

Reply:
@citizen428 Yep - your assumption is spot on! Updated it to `parse_context` 👍.